tests/app/main/views/test_user_profile.py

```diff
@@ -37,7 +37,7 @@ def test_overview_page_shows_disable_for_platform_admin(
     client_request.login(platform_admin_user)
     page = client_request.get('main.user_profile')
     assert page.select_one('h1').text.strip() == 'Your profile'
-    disable_platform_admin_row = page.select_one('#disable-platform-admin')
+    disable_platform_admin_row = page.find_all(class_='govuk-summary-list__row')[-1]
```

Comment:
I prefer CSS-style selectors:
```suggestion
    disable_platform_admin_row = page.select('.govuk-summary-list__row')[-1]
```

Reply:
I've updated this (and the other similar test line) and rebased the change in